samples/Store/Integration/AutoDataAttribute.fs

```diff
@@ -2,6 +2,22 @@
 
 open System
 
+open Domain
+open FsCheck.FSharp
+open FSharp.UMX
+
+module Arb =
+    let generate<'t> = ArbMap.defaults |> ArbMap.generate<'t>
+type FsCheckGenerators =
+    static member SkuId = Arb.generate |> Gen.map SkuId |> Arb.fromGen
+    static member ContactPreferencesId =
+        Arb.generate<Guid>
+        |> Gen.map (fun x -> sprintf "%s@test.com" (x.ToString("N")))
+        |> Gen.map ContactPreferences.ClientId
+        |> Arb.fromGen
+    static member RequestId = Arb.generate<Guid> |> Gen.map (fun x -> RequestId.parse %x) |> Arb.fromGen
+
+
```

Comment:
can this line move to L10 and put a copy before L11

Reply:
You want two spaces between the Arb an FsCheckGenerators module?

Reply:
No, a blank after the `module` line, and one between modules